public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -262,6 +267,11 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
     }
     if (this.searchText.value.length > 0) {
       this.questions = this.questions.filter(this.searchQuestion, this);
+      gtag('set', 'page', window.location.href);
```

Comment:
Could you double check whether you can remove this? I would guess this would be getting propagated automatically.

Reply:
Page is being auto tracked irrespective of this. thanks.